src/layout/index.js

```diff
@@ -0,0 +1 @@
+export { default as Layout } from './layout.js'
```

Comment:
With #3 being merged, I think we should move `src/layout` to `src/app/layout` as it's being used by the app wrapper only, it's not a shared component (if it was, then it should be in `src/shared/layout`)